src/components/BlockRenderer/BlockRenderer.jsx

```diff
@@ -23,20 +23,7 @@ function BlockRenderer(props) {
     return <ViewBlock {...props} detached onChangeBlock={() => {}} />;
   }
   if (edit) {
-    return (
```

Comment:
We can try it out, in the beginning I think I did this for be fully "detached" from the engine. But I see that might have undesired side effects.